common/src/main/resources/graphql/schemas/ObservationDB.graphql

```diff
@@ -131,7 +131,7 @@ input CreateSiderealInput {
   ra: RightAscensionInput!
   dec: DeclinationInput!
   epoch: EpochString
-  properVelocity: ProperVelocityInput
+  properMotion: ProperMotionInput
```

Comment:
I hope you updated this file by using the shell script.

Reply:
Please expand 😄

Reply:
There's a `./fetchODBSchema.sh` that calls the API via instrospection to get the schema and updates the file in the project.